### ISSUE TITLE
New optional properties

### ISSUE THREAD

**Issue by ronm123** [thexerteproject/xerteonlinetoolkits#589]
This is perhaps linked to #588 but it would be good to give users control over the following in all project templates without the need to know/add css. We already have some of this in the Bootstrap template:
LO properties:
Header background colour (including transparent)
Page background colour (including transparent)
Footer background colour (including transparent)
Page properties:
Page background colour (including transparent)
I'm sure there are others too?

**Comment by JohnSmith-LT** [comment]
Hi @FayCross @ronm123 do either of you know if these are all done or any are still outstanding and required?

**Comment by FayCross** [comment]
I don’t think any of these exist in the XOT template. In Bootstrap template you can customise header & footer colours but anything else would have to be done with a theme or css. I don’t think it’s particularly high priority.

**Comment by ronm123** [comment]
@JohnSmith-LT @FayCross Yeah I agree this is not high priority. I think it might still be desirable at some point but as Fay says it hasn't existed in the XOT template although I seem to recall we had the ability to do this from the authoring interface in the Flash days! ;-) Might be thinking about the developer tool rather than the xot editor though.
